docs/open-source-project/release-notes/tizen-5-5-m1.md

```diff
@@ -0,0 +1,631 @@
+- Input service framework
+  - Added IME API set getting key code and visibility state of candidate.
+
+
+## Multimedia framework
+
+### New and changed features
+
+- MediaTool
+  - API set to support audio channel mask has been added.
+- Media Player
+  - Audio Offloading support has been added.
+  - Some display type has been removed.
+  - API set for progressive download has been removed.
+  - player_get_streaming_download_progress pre-condition has been extended.
+  - The range of parameter values of buffering related function has been changed and the value of -1 has been allowed.
+  - Support for controlling the audio pitch has been added.
+- Media Controller
+  - Management of content season, episode, and resolution metadata has been added.
+  - Support for getting and counting playlists has been updated.
+  - Support for counting search conditions has been added.
+- Image Util
+  - image_util_calculate_buffer_size() has been deprecated.
+  - Sync transform API set (convert, crop, resize, and rotate) has been added.
+- Thumbnail Util
+  - Extract thumbnails even if requested size exceeds the source size.
+- Camera
+  - Change buffer sharing mechanism to improve buffer protection.
+  - Support user allocated buffer for zero copy preview.
+- Recorder
+  - Change buffer sharing mechanism to improve buffer protection.
+- Audio IO
+  - Support for applying VoIP latency in case of VoIP audio type.
+- PulseAudio
+  - Support for triggering audio ducking effect by recording stream also.
+  - Support for applying volume type to loopback stream.
+  - Support for getting the PID of the latest stream.
+  - Support for individual volume has been added.
+- Audio HAL
+  - API to set volume ratio to control H/W volume according to each stream type has been added.
+- C# API
+  - Common
+    - Deprecated P/Invoke functions have been changed to recommended functions.
+  - AudioManager
+    - API to check whether any stream from the current AudioStreamPolicy is actually using the device has been added.
+  - ImageUtil
+    - API set to calculate buffer size has been deprecated.
+  - MediaContent
+    - API set to control storage information has been deprecated.
+    - The possibility of MediaContent DB inconsistency has been removed.
+  - MediaTool
+    - API set to set max bps and duration of MediaTool has been added.
+  - MediaPlayer
+    - API set to control AudioPitch has been added.
+    - The precondition of GetDownloadProgress API has been changed.
+    - Description of Preparing state restriction has been added.
+    - The range and default value related BufferingTime have been changed.
+  - Metadata
+    - API to set composer has been added.
+
+### Fixes
+
+- Media Content
+  - media_info_insert_to_db() returns the updated media information. In the past, this API returns the media information without verifying that it was up-to-date.
+  - media_face_create() support only for the images.
```

Comment:
Suggest rephrasing to:

media_face_create() support only for the images has been added.

Reply:
will accept